openedx/core/djangoapps/catalog/utils.py

```diff
@@ -573,6 +573,29 @@ def get_course_run_details(course_run_key, fields):
     return course_run_details
 
 
+def is_course_run_in_program(course_run_key, program):
```

Comment:
Why do we need this function if we are not using this yet? Will be usage of this function come in later?